fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -4936,4 +4938,15 @@ public LogicalPlan visitShowCollation(ShowCollationContext ctx) {
         }
         return new ShowCollationCommand(wild);
     }
+
+    @Override
+    public LogicalPlan visitShowTableCreation(ShowTableCreationContext ctx) {
+        String dbName = null;
+        if (ctx.database != null) {
+            List<String> nameParts = visitMultipartIdentifier(ctx.database);
+            dbName = nameParts.get(0); // only one entry possible
+        }
+
+        return new ShowTableCreationCommand(dbName, ctx.STRING_LITERAL().getText());
```

Comment:
ctx.STRING_LITERAL() is not always true defined by DorisParser.g4. Pls check this behavior and add some cases

Reply:
Addressed. Please review. Also this command was supported earlier (for iceberg). Looks like the support has been removed later. Now the command just returns ok,.